postgres/adhoc_db.SQL

```diff
@@ -0,0 +1,47 @@
+CREATE TABLE candy(
+    candy_id INT PRIMARY,
+    name VARCHAR(20),
+);
+
+INSERT INTO candy(candy_id, name)
+Values (candy_id, name);
+"candy_id"	"candy_name"
+1	"Skittles"
+2	"M&M"
+3	"100 Grand"
+4	"Sour Patch Kids"
+5	"Swedish Fish"
+6	"Kit-Kat"
+7	"Snickers"
+8	"Starburts"
+9	"Milk Duds"
+10	"Nerds"
```

Comment:
This isn't sql so should either be commented out or removed, preferable the former, since these values should be inside the SQL commands themselves.